cfme/base/ui.py

```diff
@@ -62,3 +107,612 @@ class Dashboard(CFMENavigateStep):
+    VIEW = ConfigurationView
+    prerequisite = NavigateToAttribute('appliance.server', 'Configuration')
+
+    def step(self):
+        self.view.accordions.diagnostics.tree.click_path(self.obj.settings_string)
+
+
+@navigator.register(Region, 'DiagnosticsZones')
+class RegionDiagnosticsZones(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.zones.select()
+
+
+@navigator.register(Region, 'RolesByServers')
+class RegionDiagnosticsRolesByServers(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.rolesbyservers.select()
+
+
+@navigator.register(Region, 'ServersByRoles')
+class RegionDiagnosticsServersByRoles(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.serversbyroles.select()
+
+
+@navigator.register(Region, 'Servers')
+class RegionDiagnosticsServers(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.servers.select()
+
+
+@navigator.register(Region, 'Database')
+class RegionDiagnosticsDatabase(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.database.select()
+
+
+@navigator.register(Region, 'OrphanedData')
+class RegionDiagnosticsOrphanedData(CFMENavigateStep):
+    VIEW = RegionDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.orphaneddata.select()
+
+
+# ################################## ZONE NAVS ##############################
+
+
+class ZoneForm(ConfigurationView):
+    name = Input(name='name')
+    description = Input(name='description')
+    smartproxy_ip = Input(name='proxy_server_ip')
+    ntp_server_1 = Input(name='ntp_server_1')
+    ntp_server_2 = Input(name='ntp_server_2')
+    ntp_server_3 = Input(name='ntp_server_3')
+    max_scans = BootstrapSelect("max_scans")
+    username = Input(name='userid')
+    password = Input(name='password')
+    verify = Input(name='verify')
+
+    cancel_button = Button('Cancel')
+
+
+class ZoneListView(ConfigurationView):
+    configuration = Dropdown('Configuration')
+    table = Table('//div[@id="settings_list"]/table')
+
+
+# Zone Details #
+class ZoneDetailsView(ConfigurationView):
+    configuration = Dropdown('Configuration')
+
+
+@navigator.register(Zone, 'Details')
+class ZoneDetails(CFMENavigateStep):
+    VIEW = ZoneDetailsView
+
+    prerequisite = NavigateToAttribute('appliance.server.zone.region', 'Zones')
+
+    def step(self):
+        view = self.create_view(ZoneListView)
+        rows = view.table.rows((1, re.compile(r'Zone\s?\:\s?{}'.format(
+            self.obj.description))))
+        for row in rows:
+            row.click()
```

Comment:
Potential for hidden bugs here - if more than 1 row matches, we'll only ever click the first match. I'd like to at least log the fact that there were multiple matches.